pkg/interface/src/App.js

```diff
@@ -70,6 +73,10 @@ class App extends React.Component {
     this.api.local.setDark(this.themeWatcher.matches);
     this.themeWatcher.addListener(this.updateTheme);
     this.api.local.getBaseHash();
+    Mousetrap.bind(['command+l', 'ctrl+l'], (e) => {
+      e.preventDefault();
+      this.api.local.setOmnibox();
```

Comment:
ctrl-L works on safari, but cmd-L does not. I think safari may intercept the cmd-L before we see it, so this might have to be changed
cc: @jyng @g-a-v-i-n

Reply:
Blech. https://stackoverflow.com/questions/37371508/disabling-keyboard-shortcuts-using-javascript-not-working-in-safari

Reply:
Ping @g-a-v-i-n and @jyng once more — Cmd+L is intercepted on Safari but Ctrl+L isn't. Do you have additional suggestions for what we can use or are you content with falling back to Ctrl?

Reply:
I think Ctrl+L is better than Cmd+L, but I also don't think we should show the shortcut on the button. Except maybe in an onboarding flow??

Reply:
i would consider using a lib to smooth this over https://github.com/jaywcjlove/hotkeys

Reply:
Wouldn't Mousetrap do this just as well? https://medium.com/@subwaymatch/comparing-keyboard-shortcut-libraries-in-react-mousetrap-vs-hotkeys-634877b4af9e

Reply:
I can see if stoppropagation would help this as well ...

Reply:
mousetrap may not deal with edgecases a purpose-built library might.

Reply:
Tested, hotkeys has the same issue.